wasm/src/ledger/transition.rs

```diff
@@ -176,6 +176,13 @@ impl Transition {
         Group::from(self.0.tpk())
     }
 
+    /// Get the transition view key of the transition.
+    pub fn tvk(&self, view_key: &ViewKey) -> Field {
+        let tpk = self.tpk();
+        let tvk = tpk.scalar_multiply(&view_key.to_scalar()).to_x_coordinate();
+        tvk
```

Comment:
We should make it a bit more concise
```suggestion
        tpk.scalar_multiply(&view_key.to_scalar()).to_x_coordinate()
```